pkgs/desktops/gnome/default.nix

```diff
@@ -240,6 +240,8 @@ lib.makeScope pkgs.newScope (self: with self; {
 
   gnome-applets = callPackage ./misc/gnome-applets { };
 
+  gnome-epub-thumbnailer = callPackage ./misc/gnome-epub-thumbnailer { };
```

Comment:
Could you please move this to top-level? We are trying to minimize the use of `gnome` package set.